TODO

```diff
@@ -1,3 +1,2 @@
-* Signature verification for old style timestamps
-* CLI
-* MSI files
+* Move osslsigncode to own module
+* Do PE signing with pure python code
```

Comment:
+1. My local tox run failed, probably because osslsigncode 1.7.1 on mac isn't behaving as expected. Maybe we should document the known-working versions?